lib/Redis/ScriptCache.pm

```diff
@@ -1,15 +1,18 @@
 package Redis::ScriptCache;
 use strict;
 use warnings;
+use 5.10.0;
 
-our $VERSION = '0.04';
+our $VERSION = '0.05';
 
 use File::Basename;
 use File::Spec qw(
     catdir
     splitdir
 );
 use Carp;
+use Digest::SHA1 'sha1_hex';
+use Try::Tiny;
```

Comment:
Personally, I'm not a fan of Try::Tiny. That's for two and a half reasons:

- I know how it works. ;)
- It's slow (which isn't a huge deal hear since you'll do a network round-trip for most operations, but it's good sanitation to realize anyway).
- It tends to introduce subtle bugs wrt. scoping and variable life cycles.

This being said, if you're more comfortable with it, don't let me stop you. :)